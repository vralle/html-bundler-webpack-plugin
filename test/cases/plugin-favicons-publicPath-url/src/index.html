<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link href="./mario.png" rel="icon" />
  <link href="./style.css" rel="stylesheet" />
  <script src="./main.js" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>