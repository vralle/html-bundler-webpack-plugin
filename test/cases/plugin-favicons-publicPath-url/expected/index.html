<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link rel="icon" type="image/png" sizes="16x16" href="https://cdn.com/v1/favicons/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://cdn.com/v1/favicons/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="48x48" href="https://cdn.com/v1/favicons/favicon-48x48.png">
<link rel="icon" type="image/x-icon" href="https://cdn.com/v1/favicons/favicon.ico">
<link rel="manifest" href="https://cdn.com/v1/favicons/manifest.webmanifest">
<meta name="application-name" content="My App">
<meta name="mobile-web-app-capable" content="yes">
<meta name="theme-color" content="#fff">
  <link href="https://cdn.com/v1/style.bundle.css" rel="stylesheet" />
  <script src="https://cdn.com/v1/main.bundle.js" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>